actors/builtin/sector.go

```diff
@@ -1,12 +1,10 @@
-package abi
+package builtin
```

Comment:
It's hard to find a good spot for this. It's really miner policy, but used by the power actor. Since miner depends on power already, anything used by both needs to be in power, or builtin etc.

Reply:
We could introduce a new "sector" package, but this seems fine.